src/js/form-builder.js

```diff
@@ -927,10 +929,15 @@
           id: lastID + '-edit',
           className: 'toggle-form btn icon-pencil',
           title: opts.messages.hide
+        }),
+        copyBtn = utils.markup('a', opts.messages.copyButton, {
+          id: lastID + '-copy',
+          className: 'copy-button btn icon-copy',
+          title: opts.messages.copyButtonTooltip
         });
 
       var liContents = utils.markup(
-        'div', [toggleBtn, delBtn], { className: 'field-actions' }
+        'div', [toggleBtn, copyBtn, delBtn], { className: 'field-actions' }

```

Comment:
Should we change order of action buttons? I think going from most used to least used from right to left makes sense but that would mean moving the delete button also. What are your thoughts? I ask mainly because I keep cloning fields when i mean to edit them.